### PR TITLE
Survey Form: Features form content

### PR DESCRIPTION
**Pull request by lily4178993**
In this branch, I have:

- Added the HTML & CSS files
- Added the README file
- Added the screenshot of the final result

### CODE REVIEW

**Comment by lily4178993** [comment]
You have to set up all the [linters](https://github.com/microverseinc/linters-config/tree/master/html-css) required in your project

File: README.md [hunk 1]
```diff
@@ -1,2 +1,176 @@
...
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+<br/><br/>
+
+<!-- Features -->
+
+### <b>Key Features</b> <a name="key-features"></a>
+
+- **Linters** have been set up in order to track possible errors from the HTML and CSS files
+- **The stylesheet file** has been linked externaly
```
Comment (lily4178993) [inline README.md:71]: You have not properly set the required [linters](https://github.com/microverseinc/linters-config/tree/master/html-css) that you have mentionned in your README file

**Review by lily4178993 — COMMENTED** [review]
Hi @lily4178993,

You've done a very good job so far, but a few things still need to be worked on before moving on to the next project, but you're so close!

almost-there

## Highlights

❎ You have not set up the required linters.
✅ Correct use of GitHub Flow.
✅ Nice README file.


## Required Changes ♻️

_Check the comments under the review_.

------
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @lily4178993  in your question so I can receive the notification.


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s). gif_

File: README.md [hunk 2]
```diff
@@ -1,2 +1,176 @@
...
+<br/><br/>
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+<br/><br/>
+
+<!-- Features -->
+
+### <b>Key Features</b> <a name="key-features"></a>
+
+- **Linters** have been set up in order to track possible errors from the HTML and CSS files
```
Comment (lily4178993) [inline README.md:70]: You did not set up the required linters that you mentioned in your README file.
Follow this [link](https://github.com/microverseinc/linters-config/tree/master/html-css) to configure the linters

File: README.md [hunk 3]
```diff
@@ -1,2 +1,176 @@
...
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+<br/><br/>
+
+<!-- LIVE DEMO -->
+
+## <b>Live Demo 🚀</b> <a name="live-demo"></a>
+<hr />
+
+You can see the live demonstration of this project [here]().
```
Comment (lily4178993) [inline README.md:81]: Please, review the link to your live demo 🚀

**Review by nedjwab — APPROVED** [review]
# 🟢 Project Approved! 🏆

## Hi  @lily4178993 👋

Your project is complete! ✔️✔ , I loved your work, it looks great 🔥🔥🔥🔥 🏁

Your code was very professional 💯

There is nothing else to say other than... _it's time to merge it_ :shipit:
Congratulations! 🎉

 
## Highlights 🌟

- ✔️ Descriptive commmit messsages
- ✔️ Descriptive PR description
- ✔️ No linter Errors

- The code is top-notch! 💯 and have a huge potential.

Cheers and Happy coding!👏👏👏

## Important Notes 📝

- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.


_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by lily4178993** [comment]
Thank you @nedjwab for the review!

**Pull request merged**